### PR TITLE
Extra option to mount demo datasets into engine container.

### PR DESCRIPTION
**Pull request by siyuan0322**
This PR will allow user to mount the demo dataset bucket into engine container in k8s mode.

`sess = graphscope.session(with_demo_dataset=True)` will mount the `graphscope/dataset` directory to `/dataset` path in container.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -245,13 +245,32 @@ The figure shows the flow of execution in the cluster mode. When users run code
...
 
-To use GraphScope in a distributed setting, we need to establish a session in a python interpreter. 
+To use GraphScope in a distributed setting, we need to establish a session in a python interpreter.
+
+For convenience, we provide several demo datasets, and an option `with_demo_dataset` to mount the dataset in the graphscope cluster. The datasets will located in `/dataset`.
+
+```python
+import graphscope
+
+sess = graphscope.session(with_demo_dataset=True)
```
Comment (yecol) [inline README.md:255]: mount_dataset="/dataset"

File: README.md [hunk 2]
```diff
@@ -245,13 +245,32 @@ The figure shows the flow of execution in the cluster mode. When users run code
 
 ### Creating a session
 
-To use GraphScope in a distributed setting, we need to establish a session in a python interpreter. 
+To use GraphScope in a distributed setting, we need to establish a session in a python interpreter.
+
+For convenience, we provide several demo datasets, and an option `with_demo_dataset` to mount the dataset in the graphscope cluster. The datasets will located in `/dataset`.
```
Comment (yecol) [inline README.md:250]: ... The datasets will mount to `~/dataset` in the pods as default, or you can assign any other location. If you want to use your own data on k8s cluster, please refer to [this](#).

File: README.md [hunk 3]
```diff
@@ -245,13 +245,32 @@ The figure shows the flow of execution in the cluster mode. When users run code
...
+
+For convenience, we provide several demo datasets, and an option `with_demo_dataset` to mount the dataset in the graphscope cluster. The datasets will located in `/dataset`.
+
+```python
+import graphscope
+
+sess = graphscope.session(with_demo_dataset=True)
+```
+
+<details>
```
Comment (yecol) [inline README.md:258]: remove L258-268 to docs.

**Review by yecol — CHANGES_REQUESTED** [review]

**Comment by siyuan0322** [comment]
Renamed parameters according to comments and also updated the README.md and README-zh.md

**Review by sighingnow — APPROVED** [review]

**Review by lidongze0629 — APPROVED** [review]

**Pull request merged**